### DISCUSSION TITLE
Firewall rules

### DISCUSSION THREAD

**Discussion by PurpleBooth** [syself/cluster-api-provider-hetzner#271] [Ideas]
It'd be great to provision some basic firewall rules at the same time as rollout out the custer. At the moment this means I need a weird hybrid of Terraform and Cluster API.

**Comment by batistein** [comment]
Right now I don't use the hetzner firewall. Instead I use ciliums [host firewall ](https://docs.cilium.io/en/v1.12/gettingstarted/host-firewall/). But of course contributions are welcome! 
Maybe it would also be interesting to gather some information beforehand what kind of rules would need to be set.
Thank you very much for kicking off the discussion! :smile:

**Comment by PurpleBooth** [comment]
I was more thinking of some resources to make it provisionable. You know I never even considered switching to the host firewall thing cilium does. I should totally use that as my approach, thanks!
